src/plugins/data/public/search/session/sessions_mgmt/lib/get_expiration_status.ts

```diff
@@ -34,11 +34,14 @@ export const getExpirationStatus = (config: SearchSessionsConfigSchema, expires:
     const expiresInHours = Math.floor(durationToExpire.asHours());
 
     toolTipContent = i18n.translate('data.mgmt.searchSessions.status.expiresSoonInHours', {
-      defaultMessage: 'This session expires in {numHours} hours',
+      defaultMessage: `This session expires in {numHours} ${
+        expiresInHours <= 1 ? 'hour' : 'hours'
+      }`,
+      // I'm trying to use a template literal to determine if 'hour' or 'hours' should be displayed based on expiresInHours.
       values: { numHours: expiresInHours },
     });
     statusContent = i18n.translate('data.mgmt.searchSessions.status.expiresSoonInHoursTooltip', {
-      defaultMessage: '{numHours} hours',
+      defaultMessage: `{numHours} ${expiresInHours <= 1 ? 'hour' : 'hours'}`,
```

Comment:
yes, this solution would work, and output the desired hours/hours 👍 
But it would break our i18n, extracting those defaultMessages, which should be 1 string 😢 
Luckily there's help, a way to provide syntax for pluralization in the defaultMessage 🎉 

https://github.com/elastic/kibana/blob/3c8b26b53dbba9396511bd7a258818afe6cce878/src/plugins/console/public/application/components/settings_modal.tsx#L41-L43

https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#pluralization